probe/http/http.go

```diff
@@ -155,6 +160,7 @@ func (h *HTTP) DoProbe() (bool, string) {
 		return false, fmt.Sprintf("HTTP Status Code is %d. It missed in %v", resp.StatusCode, h.SuccessCode)
 	}
 
+	h.ExportMetrics(resp)
```

Comment:
Question: Do we just export metrics for the valid situation?

Reply:
we need to consider the failed situation.